jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -34,6 +37,21 @@
  */
 public class LayerEntry {
 
+  /**
+   * Stringifies {@code path} in Unix form. The path must be absolute.
+   *
+   * @param path the path
+   * @return the string form of the absolute path
+   */
+  private static String toAbsoluteUnixPath(Path path) {
```

Comment:
`toUnixPath` sounds better to me. I mistakenly thought `toAbsolute...` would convert `path` to absolute first.